.github/workflows/bb-fork-forge-test.yml

```
@@ -0,0 +1,176 @@
+      FOUNDRY_PROFILE: ${{ inputs.foundry-profile }}
+      MAINNET_RPC_URL: "${{ secrets.MAINNET_RPC_URL }}"
+      ETHERSCAN_API_KEY: "${{ secrets.ETHERSCAN_API_KEY }}"
+      BUILDBEAR_BASE_URL: "${{ secrets.BUILDBEAR_BASE_URL }}"
+    runs-on: "ubuntu-latest"
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          submodules: recursive
+
+      - name: Install Rust
+        uses: actions-rs/toolchain@v1
```

actionlint (1.7.10)

[error] the runner of "actions-rs/toolchain@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)